Modules/Setup.bootstrap.in

```diff
@@ -30,6 +30,7 @@ _weakref _weakref.c
 _abc _abc.c
 _functools _functoolsmodule.c
 _locale _localemodule.c
+_opcode _opcode.c
```

Comment:
You should remove the `PY_STDLIB_MOD_SIMPLE` def for `_opcode` from `configure.ac` now; it's not needed anymore.